q2_types/feature_table/_transformer.py

```diff
@@ -12,26 +12,36 @@
 import pandas as pd
 import qiime
 
-from . import BIOMV1Format
+from . import BIOMV1Format, BIOMV210Format
 from ..plugin_setup import plugin
 
 
+def _get_generated_by():
+    return 'qiime %s' % qiime.__version__
+
+
 @plugin.register_transformer
 def _1(data: biom.Table) -> BIOMV1Format:
+    # TODO: issue a warning about limited support for this format
```

Comment:
Yup, that makes sense, I am just wondering if it would make more sense to raise those kinds of warnings in biom (so that things are centralized there), but then we make sure to let them bubble up in the plugin here (that way we don't wind up with a situation where we have conflicting recommendations on biom format --- one version here, another version there, etc.).